frontend/src/components/molecules/SustainabilityGoalsList.tsx

```diff
@@ -0,0 +1,57 @@
+import { Center, SimpleGrid } from '@chakra-ui/react';
+import React, { useEffect, useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { Link } from 'react-router-dom';
+import { getSustainabilityGoals } from '../../api/ontologies';
+import { mapIdToNode } from '../../common/node';
+import { setError } from '../../state/reducers/apiErrorReducer';
+import { selectNode } from '../../state/reducers/ontologyReducer';
+import { Node, SustainabilityGoal } from '../../types/ontologyTypes';
+import IconContainer from '../atoms/IconContainer';
+
+// Initial node displayed on the frontpage
+const sustainabilityNode: Node = {
+  prefix: {
+    prefix: 'SDG',
+    iri: 'http://www.semanticweb.org/aga/ontologies/2017/9/SDG#',
+  },
+  name: 'SDG',
+  id: 'http://www.semanticweb.org/aga/ontologies/2017/9/SDG#SDG',
+};
+
+const SustainabilityGoalsList: React.FC = () => {
+  const [sustainabilityGoals, setSustainabilityGoals] = useState<SustainabilityGoal[]>();
```

Comment:
```suggestion
  const [sustainabilityGoals, setSustainabilityGoals] = useState<Array<SustainabilityGoal>>();
```
to match the format we've gone with in the rest of the codebase

Reply:
Also as we do have a type that is `SustainabilityGoals` then it should be used here as well.